### PR TITLE
fetch unstaking tokens

### LINKED ISSUE #176: Show the unstaking tokens
### Describe the bug

Once the user has unstaked the rewards, they are locked for 7 days. This locked tokens are not shown on the UI. 

### Reproduction

1. Go to /collators
2. Click on unbond to remove the delegation
3. If successfull, the delegation is removed from that collator
4. User cant see the locked tokens in the `AMPE unboarding`

### Expected behavior

Once the tokens are unbonded the locked tokens should be shown under `AMPE unboarding`

### Screenshots

image

### Desktop (please complete the following information):

- OS: MACOS
- Browser: Brave

### PR DESCRIPTION
**Pull request by gonzamontiel**
Closes #176

### CODE REVIEW

File: src/pages/collators/CollatorRewards.tsx [hunk 1]
```diff
@@ -45,10 +46,18 @@ function CollatorRewards() {
...
       const { data: balance } = await api.query.system.account(walletAccount?.address);
-      return balance.free.sub(balance.miscFrozen).toString();
+      setUserAvailableBalance(balance.free.sub(balance.miscFrozen).toString());
+    };
+    const fetchUnstaking = async () => {
+      if (!api || !walletAccount) {
+        return '0.00';
+      }
+      const unstakingData = await api.query.parachainStaking.unstaking(walletAccount?.address);
+      unstakingData.forEach((n) => setUnbording(nativeToFormat(parseInt(n.toString()), tokenSymbol)));
```
Comment (ebma) [inline src/pages/collators/CollatorRewards.tsx:56]:
```suggestion
      unstakingData.forEach((n) => setUnboarding(nativeToFormat(parseInt(n.toString()), tokenSymbol)));
```

**Review by ebma — APPROVED** [review]
Looks good overall but I still find the terminology confusing.
Why are we saying 'Unbond' in one case and 'unboard' in another. Shouldn't we make that consistent? It's the same thing afterall @prayagd?
Also, Kilt is using a different terminology, ie. they phrase it as 'unlocking unstaked tokens', see [here](https://docs.kilt.io/docs/participate/staking/unlock-unstaked).

**Comment by prayagd** [comment]
> Why are we saying 'Unbond' in one case and 'unboard' in another. Shouldn't we make that consistent? It's the same thing afterall @prayagd?

@ebma Yes they are the same, its a typo. @gonzamontiel Can we make it unbonding? I find the KILT phrase too long.

**Comment by gonzamontiel** [comment]
> Looks good overall but I still find the terminology confusing. Why are we saying 'Unbond' in one case and 'unboard' in another. Shouldn't we make that consistent? It's the same thing afterall @prayagd? Also, Kilt is using a different terminology, ie. they phrase it as 'unlocking unstaked tokens', see [here](https://docs.kilt.io/docs/participate/staking/unlock-unstaked).

Yeah I agree, it's a nice chance for changing that for good. I like 'unlocking unstaked tokens', it is clearer.

**Comment by gonzamontiel** [comment]
> > Why are we saying 'Unbond' in one case and 'unboard' in another. Shouldn't we make that consistent? It's the same thing afterall @prayagd?
> 
> @ebma Yes they are the same, its a typo. @gonzamontiel Can we make it unbonding? I find the KILT phrase too long.

All this time it has been a typo? lol. I will change it then.

**Pull request merged**